docker-compose_swarm.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+services:
+  postgres:
+    image: "postgres:${POSTGRES_VERSION:-17.0-bookworm}"
```

Comment:
can this be pointing to latest?

Reply:
I afraid that newer version can contain some breaking changes. Also it seems like a well-known anti-pattern.
There are some articles about it:
https://www.docker.com/blog/docker-best-practices-using-tags-and-labels-to-manage-docker-image-sprawl/
https://medium.com/@mehar.chand.cloud/docker-10-anti-patterns-what-to-avoid-980fa13d8951#c3f1
https://stackoverflow.com/a/72891342